### DISCUSSION TITLE
Error with pruning off

### DISCUSSION THREAD

**Discussion by Pkaps25** [ChaissonLab/danbing-tk#3] [Q&A] [answered]
Hello, when building the RPGG with `pruning` set to `True` in the config, the pipeline runs successfully. However, with when set to `False`, an error occurs. Please let me know if any other info would be helpful; I've attached the error log.
[2021-01-18T175829.471764.snakemake.txt](https://github.com/ChaissonLab/danbing-tk/files/5832707/2021-01-18T175829.471764.snakemake.txt)

**Comment by joyeuxnoel8** [comment, marked as answer]
I fixed the wrong `kmerpref` in rule `GenPanGenomeGraph`. Let me know how it works. Thanks!

**Reply by Pkaps25** [reply]
Thank you, it worked!
